rbac/providers/common/provider_transforms.py

```diff
@@ -45,10 +47,11 @@
     "usage_location": {"azure": "usageLocation", "ldap": None},
     "user_principal_name": {"azure": "userPrincipalName", "ldap": "userPrincipalName"},
     "user_type": {"azure": "userType", "ldap": None},
+    "relationship_id": {"azure": "id", "ldap": "distinguishedName"},
```

Comment:
There is still some confusion around this as id and distinguishedName are apparently not equivalent across providers.

Reply:
It doesn’t get mapped back given it doesn’t exist in the scheme. Thus, it doesn’t matter.